### DISCUSSION TITLE
SD3 with DDIM does not work?

### DISCUSSION THREAD

**Discussion by Pengchengpcx** [huggingface/diffusers#9349] [Q&A]
I tried to use stable diffusion 3 with DDIMscheduler but it does not work. I am wondering if DDIM can be used for SD3 ? Here is what I used. I think the default FlowmatchEuler is also an ODE solver as the DDIM. I also tried different betas but I cannot produce good images with DDIM.

`ddim_sch = DDIMScheduler(beta_start=0.00085, beta_end=0.012, beta_schedule="scaled_linear", clip_sample=False, set_alpha_to_one=False, steps_offset=1)`

`pipe = StableDiffusion3Pipeline.from_pretrained("/apdcephfs_cq8/share_2992679/ckpt/stable-diffusion-3-medium-diffusers/", torch_dtype=torch.float16, local_files_only=True)`

`pipe.scheduler = ddim_sch`

**Comment by asomoza** [comment]
cc: @yiyixuxu
